zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/appliers/EventAppliers.java

```diff
@@ -445,8 +445,11 @@ private void registerDecisionRequirementsAppliers(final MutableProcessingState s
         new DecisionRequirementsDeletedApplier(state.getDecisionState()));
   }
 
-  private void registerDecisionEvaluationAppliers() {
-    register(DecisionEvaluationIntent.EVALUATED, NOOP_EVENT_APPLIER);
+  private void registerDecisionEvaluationAppliers(final MutableProcessingState state) {
+    register(
+        DecisionEvaluationIntent.EVALUATED,
+        1,
+        new DecisionEvaluationV1Applier(state.getUsageMetricState()));
```

Comment:
💭 I'm thinking we might still need to add a V2 and leave V1 NOOP.

Reason: When the engine is replaying older events, there will be V1 EVALUATED events without the usage metrics EXPORTED event before. This means during replay it will call the applier and record eDI metric but there won't be an active bucket yet.
We can't track events that were produced before our feature.

CI failures might confirm it, let's keep an eye on them. (CI is quite flaky right now due to my PR)